documentation/source/ObjectDetection.md

```diff
@@ -76,12 +175,58 @@ Box coordinates are in absolute (pixel) units.
 Visualization of the model predictions is a very important part of the training process for any computer vision task. 
 By visualizing the predicted boxes, developers and researchers can identify errors or inaccuracies in the model's output and adjust the model's architecture or training data accordingly.
 
-SuperGradients provide an implementation of [DetectionVisualizationCallback](https://docs.deci.ai/super-gradients/docstring/training/utils.html#training.utils.callbacks.callbacks.DetectionVisualizationCallback). 
-You can use this callback in your training pipeline to visualize predictions during training. For this, just add it to `training_hyperparams.phase_callbacks` in your yaml.
-During training, the callback will generate a visualization of the model predictions and save it to the TensorBoard or Weights & Biases depending on which logger you
-are using (Default is Tensorboard). 
+#### Extreme Batch Visualization during training
+
+SuperGradients provide an implementation of [ExtremeBatchDetectionVisualizationCallback](https://docs.deci.ai/super-gradients/docstring/training/utils.html#src.super_gradients.training.utils.callbacks.callbacks.ExtremeBatchDetectionVisualizationCallback). 
```

Comment:
`provide` -> `provides`

Reply:
Fixed in 4ad8fdf3843bc9b6882c329055ea664ba31ddbe1